test/Groups/matrixgroups.jl

```diff
@@ -0,0 +1,457 @@
+@testset "Oscar-GAP relationship" begin
+   F = GF(29,1)[1]
+   z = F(2)
+   G = GL(3,F)
+   @test G.X isa GapObj
+   @test isdefined(G,:X)
+   @test isdefined(G, :mat_iso)
+   @test G.mat_iso.fr(z) isa FFE
+   Z = G.mat_iso.fr(z)
+   @test GAP.Globals.IN(Z,G.mat_iso.fr.codomain)
+   @test G.mat_iso.fr(Z)==z
+   @test G.mat_iso.fr.domain==F
+   @test GAP.Globals.IsField(G.mat_iso.fr.codomain)
+   @test GAP.Globals.Size(G.mat_iso.fr.codomain)==29
+   @test GAP.Globals.IsZero(14*Z+1)
+   @test iszero(G.mat_iso.fr(GAP.Globals.Zero(G.mat_iso.fr.codomain)))
+   @test GAP.Globals.IsOne(G.mat_iso.fr(one(F)))
+   @test isone(G.mat_iso.fr(GAP.Globals.One(G.mat_iso.fr.codomain)))
+   
+   xo = matrix(F,3,3,[1,z,0,0,1,2*z+1,0,0,z+2])
+#   xg = Vector{GapObj}(undef, 3)
+#   for i in 1:3
+#      xg[i] = GapObj([G.mat_iso.fr(xo[i,j]) for j in 1:3])
+#   end
+#   xg=GAP.julia_to_gap(xg)
+   riso=Oscar.gen_ring_iso(F)
```

Comment:
As far as I can tell, you "only" test if `riso` is bijective -- but not that it is a homomorphism. I would feel much better if there were a few simple tests. I.e., for finite fields of order 4, 8, 9, 25, 27,  you could simply iterate over all pairs a,b and compare the image of `a*b` with the product of the images of `a` and `b`.

Reply:
Done it.